lib/screens/tags_screen.dart

```diff
@@ -1,17 +1,134 @@
+import 'package:finance_tracker/screens/add_tag_screen.dart';
 import 'package:flutter/material.dart';
 
-class TagsScreen extends StatelessWidget {
+class TagsScreen extends StatefulWidget {
   const TagsScreen({super.key});
 
+  @override
+  _TagScreenState createState() => _TagScreenState();
+}
+
+class _TagScreenState extends State<TagsScreen> {
+  final List<Map<String, String>> tags = [
+    {'title': 'Groceries', 'description': 'dsjflkhsdjkfhslkdfhkdshgkd'},
+    {'title': 'Utilities', 'description': 'djkshgflkghadkghfdjvnsdkn'},
+    {
+      'title': 'Entertainment',
+      'description':
+          'fdöybjndsökb iaugpnrehwgvrpvtiueiebrtbemhrawibghslegirwanötertwitbnörbösgnsrg rhgrehreg'
+    },
+  ];
+
```

Comment:
TODO als kommentar einfügen:7

// TODO: use real data instead of static test data